frontend/src/services/index.js

```diff
@@ -26,6 +28,9 @@ export const createConnection = (handler, token, roomId) => {
   socket.on('play', handler.onPlay || noop);
   socket.on('notification', handler.onNotification || noop);
   socket.on('stop', handler.onStop || noop);
+  socket.on('userlist', handler.onEvent || function(data) {
+    store.dispatch(receiveUserlist(data));
```

Comment:
not sure if this should also be a noop & use store.dispatch in the view... I guess not.